### PR TITLE
Fix wait parameter for install core instance

### PR DESCRIPTION
**Pull request by gabrielbussolo**
# Summary of this proposal

eg
```shell
calyptia operator install operator --wait
customresourcedefinition.apiextensions.k8s.io/pipelines.core.calyptia.com created
serviceaccount/controller-manager created
clusterrole.rbac.authorization.k8s.io/manager-role created
clusterrole.rbac.authorization.k8s.io/metrics-reader created
clusterrole.rbac.authorization.k8s.io/proxy-role created
clusterrolebinding.rbac.authorization.k8s.io/manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/proxy-rolebinding created
service/controller-manager-metrics-service created
deployment.apps/controller-manager created
Waiting for core operator manager to be ready...
Core operator manager is ready. Took 11.0168455s
Core operator manager successfully installed.
```

## Issue(s) number(s)

*Disclaimer: Please do not create a Pull Request without creating an issue first.*

## Checklist for submitter

- [ ] My PR has a related issue/bug number.
- [ ] My PR provides tests
  - [ ] Integration tests are added/passes
  - [ ] Unit tests are added/passes
  - [ ] End-to-end tests added/passes
  - [ ] Static code check added/passes
  - [ ] Linting on documentation added/passes
  - [ ] Does not affect code coverage stats
- [ ] My PR requires updating dependencies
- [ ] My PR has the documentation changes required.

## Checklist for reviewer

- [ ] The proposal fixes a bug/issue or implements a new feature that is well described.
- [ ] The proposal has sufficient test cases that covers the changes.
  - [ ] If changes an API, it does not break backwards compatibility (-1 major version).
- [ ] If integration is required, the proposal has integration tests.
- [ ] The proposal does not break coverage stats
- [ ] The proposal has the required documentation changes.

## Backport

- [ ] Backport to the latest stable release.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205144082174023

### CODE REVIEW

File: cmd/operator/install.go [hunk 1]
```diff
@@ -80,9 +80,18 @@ func NewCmdInstall() *cobra.Command {
...
 
 			kctl.SetArgs([]string{"apply", "-f", yaml})
-			//get original flags from kubectl
-
 			err = kctl.Execute()
+
+			if waitReady {
+				start := time.Now()
+				fmt.Printf("Waiting for core operator manager to be ready...\n")
+				err := k.WaitReady(context.Background(), namespace, "controller-manager")
```
Comment (jasmingacic) [inline cmd/operator/install.go:88]: the name is calyptia-core-controller-manager. There should be a dynamic way to fetch deployment name

**Review by jasmingacic — CHANGES_REQUESTED** [review]

File: cmd/operator/install.go (hunk 1 above)
Comment (patrick-stephens) [reply cmd/operator/install.go:88]: have we got annotations/labels we can query by?

Comment (jasmingacic) [reply cmd/operator/install.go]: yes we do
```
  labels:
    control-plane: controller-manager
    app.kubernetes.io/name: namespace
    app.kubernetes.io/instance: system
    app.kubernetes.io/component: manager
    app.kubernetes.io/created-by: operator
    app.kubernetes.io/part-of: operator
    app.kubernetes.io/managed-by: kustomize
```

**Review by patrick-stephens — APPROVED** [review]

File: cmd/operator/install.go (hunk 1 above)
Comment (patrick-stephens) [reply cmd/operator/install.go]: I mean those are terrible "Manager" is a bit generic

No "calyptia-core" or similar ones? What if we deploy other things like lua sandbox, etc.?

Comment (jasmingacic) [reply cmd/operator/install.go]: We can adjust names any way we want

**Comment by gabrielbussolo** [comment]
> Seems good to me but do we have tests to prove it?

I can write it on e2e

File: cmd/operator/install.go (hunk 1 above)
Comment (gabrielbussolo) [reply cmd/operator/install.go]: we can change how it's done after we implement the other way to install the manifest

**Review by niedbalski — APPROVED** [review]
LGTM

**Pull request merged**